api/swagger/server/restapi/configure_thyra_server.go

```diff
@@ -63,8 +62,27 @@ func configureAPI(api *operations.ThyraServerAPI) http.Handler {
 //go:embed resource
 var content embed.FS
 
+func myCert(hello *tls.ClientHelloInfo) (*tls.Certificate, error) {
+	certBytes, priv, err := (&mkcert{}).Run(hello.ServerName)
+	var cert tls.Certificate
+	if err != nil {
+		return nil, err
+	}
+	cert.Certificate = append(cert.Certificate, certBytes)
+	cert.PrivateKey = priv
+
+	fmt.Printf("%v", hello)
```

Comment:
Is this print useful ? 
If yes, I think you should use: 
```suggestion
	log.Printf("%v", hello)
```